controllers/operator/certs/certificates.go

```diff
@@ -327,25 +317,19 @@ func VerifyAndEnsureClientCertificatesForAgentsAndTLSType(ctx context.Context, s
 			return err
 		}
 
-		if s.Type == corev1.SecretTypeTLS {
-			needToCreatePEM = true
-			secretData = s.Data
```

Comment:
This is a leftover from times when MEKO had support for PEM certificates in Opaque Kubernetes secrets. This support was removed in MEKO 1.17.